default/stringtables/en.txt

```diff
@@ -1803,9 +1816,9 @@ GSETUP_BEGINNER
 Beginner Mode
 GSETUP_TURTLE
 Turtle
-GSETUP_DEFENSIVE
+GSETUP_CAUTIOUS
 Cautious
-GSETUP_MODERATE
```

Comment:
Might also change the references in UI/GalaxySetupWnd.cpp:269 and util/EnumText.cpp:40